### PR TITLE
revad: release version v1.24.0 with all the OCM new features

### PR DESCRIPTION
**Pull request by SamuAlfageme**
### Contributing a Chart / update to an existing Chart

- [x] Run `helm lint` on the chart dir.
- [x] (Update) Bump the `Chart.yaml` version before merging, to release it as a new version.
- [x] (Update) Replace the `annotations` on the `Chart.yaml` with:
  - `artifacthub.io/changes` - the changes introduced on the PR [with the appropiate format](https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations).
  - `artifacthub.io/images` - the updated tag on the `cs3org/revad` image.
- [x] (Update) If the PR includes new configurable parameters in the chart's `values.yaml`. Add documentation in the appropiate README.

### CODE REVIEW

**Comment by SamuAlfageme** [comment]
Setting as WIP until a new version of revad with https://github.com/cs3org/reva/pull/3723 gets released

File: revad/Chart.yaml [hunk 1]
```diff
@@ -23,11 +23,161 @@ keywords:
...
+    - kind: changed
+      description: Migrate the virtualViews test from Drone to GitHub Actions
+      links:
+        - name: cs3org/reva#3630
+          url: https://github.com/cs3org/reva/pull/3630
+    - kind: deprecated
+      description: Cleanup unused configs in OCM HTTP service
+      links:
+        - name: cs3org/reva#3675
+          url: https://github.com/cs3org/reva/pull/3675
```
Comment (glpatcern) [inline revad/Chart.yaml:115]:
```suggestion
```

File: revad/Chart.yaml [hunk 2]
```diff
@@ -23,11 +23,161 @@ keywords:
...
     - kind: changed
-      description: Bump revad version to v.1.21.0
+      description: Create and list OCM shares in OCS layer
```
Comment (glpatcern) [inline revad/Chart.yaml:117]:
```suggestion
      description: Incorporated support for OCM shares
```

File: revad/Chart.yaml [hunk 3]
```diff
@@ -23,11 +23,161 @@ keywords:
...
+    - kind: added
+      description: Add OCM scope and webdav endpoint
+      links:
+        - name: cs3org/reva#3691
+          url: https://github.com/cs3org/reva/pull/3691
+    - kind: changed
+      description: Revamp OCM invitation workflow
+      links:
+        - name: cs3org/reva#3611
+          url: https://github.com/cs3org/reva/pull/3611
```
Comment (glpatcern) [inline revad/Chart.yaml:175]:
```suggestion
```

File: revad/Chart.yaml [hunk 4]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: revad
 description: The Reva daemon (revad) helm chart
 type: application
-version: 1.5.0
-appVersion: v1.21.0
+version: 1.6.0
+appVersion: v1.23.0
```
Comment (glpatcern) [inline revad/Chart.yaml:6]:
```suggestion
appVersion: v1.24.0
```

File: revad/Chart.yaml [hunk 5]
```diff
@@ -23,11 +23,161 @@ keywords:
...
+      description: Bump reva(d) base image to alpine 3.17
       links:
-        - name: cs3org/reva#3524
-          url: https://github.com/cs3org/reva/pull/3524
+        - name: cs3org/reva#3703
+          url: https://github.com/cs3org/reva/pull/3703
   artifacthub.io/images: |
     - name: revad
-      image: cs3org/revad:v1.21.0
+      image: cs3org/revad:v1.23.0
```
Comment (glpatcern) [inline revad/Chart.yaml:183]:
```suggestion
      image: cs3org/revad:v1.24.0
```

File: revad/README.md [hunk 6]
```diff
@@ -36,7 +36,7 @@ The following configurations may be set. It is recommended to use `values.yaml`
...
 | `replicaCount`                             | How many replicas to run.                                                                                    | `1`                                                                                                                       |
 | `image.repository`                         | Name of the image to run, without the tag.                                                                   | [`cs3org/revad`](https://hub.docker.com/r/cs3org/revad)                                                                   |
-| `image.tag`                                | The image tag to use.                                                                                        | `v1.7.0`                                                                                                                  |
+| `image.tag`                                | The image tag to use.                                                                                        | `v1.23.0`                                                                                                                 |
```
Comment (glpatcern) [inline revad/README.md:39]:
```suggestion
| `image.tag`                                | The image tag to use.                                                                                        | `v1.24.0`                                                                                                                 |
```

**Review by glpatcern — CHANGES_REQUESTED** [review]
Prepared changes + simplified changelog for the upcoming revad version yet to be tagged.

File: revad/Chart.yaml [hunk 7]
```diff
@@ -24,10 +24,70 @@ keywords:
...
+        - name: cs3org/reva#3687
+          url: https://github.com/cs3org/reva/pull/3687
+    - kind: added
+      description: Add OCM scope and webdav endpoint
+      links:
+        - name: cs3org/reva#3691
+          url: https://github.com/cs3org/reva/pull/3691
+    - kind: changed
+      description: Revamp OCM invitation workflow
+      links:
```
Comment (glpatcern) [inline revad/Chart.yaml:83]:
```suggestion
```

File: revad/Chart.yaml [hunk 8]
```diff
@@ -24,10 +24,17 @@ keywords:
 annotations:
   artifacthub.io/changes: |
     - kind: changed
```
Comment (glpatcern) [inline revad/Chart.yaml:26]:
```suggestion
    - kind: added
```

File: revad/values.yaml [hunk 9]
```diff
@@ -2,7 +2,7 @@ replicaCount: 1
 
 image:
   repository: cs3org/revad
-  tag: v1.21.0
+  tag: v1.23.0
```
Comment (glpatcern) [inline revad/values.yaml:5]:
```suggestion
  tag: v1.24.0
```

**Review by glpatcern — APPROVED** [review]

**Comment by glpatcern** [comment]
Pending on cs3org/reva#3810 to tag a new version

**Review by mirekys — APPROVED** [review]

**Pull request merged**